.circleci/config.yml

```diff
@@ -88,6 +88,12 @@ jobs:
               echo "${DOCKER_PASSWORD}" | docker login -u="${DOCKER_USERNAME}" --password-stdin
               retry docker tag "local/socorro_app:latest" "mozilla/socorro_app:${DOCKER_TAG}"
               retry docker push "mozilla/socorro_app:${DOCKER_TAG}"
+
+              # push `latest` on master only
+              if [ "${CIRCLE_BRANCH}" == "master" ] then
```

Comment:
This is missing a `;` between the `]` and the `then`. Pretty sure that's why it failed in CircleCI.